Remove automatic :running_in_container setting

jeffkreeftmeijer opened the pull request:
Depends on https://github.com/appsignal/appsignal-agent/pull/217.

tombruijn commented on lib/appsignal/cli/diagnose.rb:
```diff
@@ -79,9 +79,8 @@ def host_information
-          return unless Appsignal::System.container?
-          puts "  Container id: #{Appsignal::System::Container.id}"
+	  print "  Running in container: "
+	  puts Appsignal::Extension.running_in_container == 1 ? "yes" : "no"
```
indenting 🐟

thijsc replied:
Since in returns a boolean already you don't need the `== 1`.

tombruijn commented on ext/appsignal_extension.c:
```diff
@@ -635,6 +639,7 @@ void Init_appsignal_extension(void) {
 
   // Event hook installation
   rb_define_singleton_method(Extension, "install_allocation_event_hook", install_allocation_event_hook, 0);
+  rb_define_singleton_method(Extension, "running_in_container", running_in_container, 0);
```
We can't return a boolean in a `running_in_container*?*` method?

thijsc replied:
You can add a wrapper method in `lib/extension.rb`:

```ruby
def running_in_container?
  running_in_container == 1
end
```

thijsc replied:
Ignore that, you can just call it `running_in_container?` here in the second argument. It already returns a boolean.

jeffkreeftmeijer replied:
Hm, I was convinced I already tried it with the question mark. 🤔 Anyway, fixed.

jeffkreeftmeijer commented:
Changed base to develop, rebased on develop and removed the agent update commit, as the current agent version in develop is newer. Tested locally with the diagnose task, this one is ready for re-review.
